Sources/SwiftSyntaxBuilder/DeclBuildables.swift.gyb

```diff
@@ -0,0 +1,83 @@
+%{
+  from gyb_syntax_support import *
+  from gyb_syntax_support.kinds import lowercase_first_word
+  from gyb_syntax_support.kinds import SYNTAX_BASE_KINDS
+  NODE_MAP = create_node_map()
+  # -*- mode: Swift -*-
+  # Ignore the following admonition it applies to the resulting .swift file only
+  
+  def syntax_buildable_child_type(type_name, syntax_kind, is_token, is_optional=False):
```

Comment:
Could you move this function to some common place in e.g. `gyb_syntax_support` that we can import instead of duplicating it?

Reply:
Should it be placed in the swift or swift-syntax repo?